login-workflow/src/components/ChangePasswordDialog/ChangePasswordDialogBase.tsx

```diff
@@ -99,6 +100,7 @@ export const ChangePasswordDialogBase: React.FC<ChangePasswordDialogProps> = (pr
                         id="current-password"
                         label={currentPasswordLabel}
                         value={currentPassword}
+                        {...currentPasswordTextFieldProps}
```

Comment:
Do we need to merge OnPasswordChange here like https://github.com/etn-ccis/blui-react-workflows/blob/dev/login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx#L142.

Same comment for other screens